src/zope/testrunner/tests/testrunner.rst

```diff
@@ -0,0 +1,71 @@
+==========
+ Overview
+==========
+
+The testrunner module is used to run automated tests defined using the
+unittest framework. Its primary feature is that it *finds* tests by
```

Comment:
Perhaps `unittest` should be a link to docs.python.org/library/unittest?  I'm not familiar with interpshinx syntax.

Reply:
I recently learned that if you put `default_role = 'obj'` in `conf.py` then if you simply put backticks around an identifier you want linked, it tends to do the right thing.